packages/client/hmi-client/src/page/data-explorer/components/tera-selected-resources-header-pane.vue

```diff
@@ -62,57 +60,42 @@ const addResourcesToProject = async (projectId: string) => {
 
 			// first, insert into the proper table/collection
 			const res = await addDocuments(body);
-			if (res && resources) {
+			if (res) {
 				const documentId = res.id;
 
 				// then, link and store in the project assets
 				const assetsType = AssetType.Publications;
-				await ProjectService.addAsset(projectId, assetsType, documentId);
+				await addAsset(projectId, assetsType, documentId);
 			}
 		}
 		if (isModel(selectedItem)) {
 			// FIXME: handle cases where assets is already added to the project
 			const modelId = selectedItem.id;
 			// then, link and store in the project assets
 			const assetsType = AssetType.Models;
-			await ProjectService.addAsset(projectId, assetsType, modelId);
+			await addAsset(projectId, assetsType, modelId);
 		}
 		if (isDataset(selectedItem)) {
 			// FIXME: handle cases where assets is already added to the project
 			const datasetId = selectedItem.id;
 			// then, link and store in the project assets
 			const assetsType = AssetType.Datasets;
 			if (datasetId) {
-				await ProjectService.addAsset(projectId, assetsType, datasetId);
+				await addAsset(projectId, assetsType, datasetId);
 			}
 		}
 	});
 };
 
-const addAssetsToProject = async (projectName) => {
+const addAssetsToProject = async (projectOption) => {
 	if (props.selectedSearchItems.length === 0) return;
 
-	let projectId = '';
-	if (projectName !== undefined && typeof projectName.value === 'string') {
-		const project = projectsList.value.find((p) => p.name === projectName.value);
-		projectId = project?.id as string;
-	} else {
-		if (!resources.activeProject) return;
-		projectId = resources.activeProject.id;
-	}
-
+	const projectId = projectOption.id ?? activeProject.value?.id;
```

Comment:
This wasn't working for me, you need to change it to `projectOption.value.id` for it to work